### PR TITLE
Fix/cython dependency

### PR DESCRIPTION
**Pull request by BalazsHoranyi**
add Cython as a primitive dependency instead of a package dependency

### CODE REVIEW

File: distil/primitives/audio_transfer.py [hunk 1]
```diff
@@ -86,6 +86,10 @@ class AudioTransferPrimitive(unsupervised_learning.UnsupervisedLearnerPrimitiveB
                     'package_uri': 'git+https://github.com/uncharted-distil/distil-primitives.git@{git_commit}#egg=distil-primitives'.format(
                         git_commit=utils.current_git_commit(os.path.dirname(__file__)),
                     )
+                }, {
+                    'type': metadata_base.PrimitiveInstallationType.PIP,
```
Comment (mitar) [inline distil/primitives/audio_transfer.py]: Sorry to bother you here more, but:
* The order is important, so this should be before you package.
* You should use `0.29.14` version. This is the one which will be fixed program-wide for next release.

Comment (BalazsHoranyi) [reply distil/primitives/audio_transfer.py]: No worries. Thank you for your help. The order and version have been updated.

**Comment by mitar** [comment]
So now there is another issue: you added your cython dependency to only one primitive, but your Python package installs multiple primitives. You generally cannot know which of your primitives is being installed, but your Python package as a whole depends on cython. But what happens if in fact some other primitive is being installed, then cython dependency will not be available. So you have two options:
* All primitives in your Python package define cython as a dependency (you can move that to some shared global variable).
* Or, you move the Python dependency which requires cython out of the dependency of the whole Python package, and make it a particular primitive's dependency. Of course in right order (cython, that special dependency, your Python package).

See example here: https://gitlab.com/datadrivendiscovery/common-primitives/blob/master/common_primitives/audio_reader.py#L56 (which does the second approach)

**Comment by BalazsHoranyi** [comment]
Opted for the first option. Should be good now?

**Comment by mitar** [comment]
I think so.

**Pull request merged**